shared/chat/manage-channels/delete-channel.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from 'react'
+import ReactDOM from 'react-dom'
+import {Text, Box, Icon, PopupMenu} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+
+const PopupHeader = ({channelName}: {channelName: string}) => {
+  return (
+    <Box
+      style={{
+        ...globalStyles.flexBoxColumn,
+        alignItems: 'center',
+        paddingLeft: globalMargins.tiny,
+        paddingRight: globalMargins.tiny,
+        paddingTop: globalMargins.small,
+        width: '100%',
+      }}
+    >
+      <Text type="BodySemibold" style={{color: globalColors.black, textAlign: 'center'}}>
+        {`Are you sure you want to delete #${channelName}?`}
```

Comment:
Why not do `Are you sure you want to delete #{channelName}?` ?

Reply:
Done!